### PR TITLE
Filter css

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me

**Review by Caroline-CD — DISMISSED** [review]
Ok for me

File: public/index.html [hunk 1]
```diff
@@ -31,6 +32,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap" rel="stylesheet">
```
Comment (LoicBrassart) [inline public/index.html]: Unindented file -> this didn't go through Prettier/ESLint, what happened ?

**Review by LoicBrassart — DISMISSED** [review]
I have a doubt about your tools, Prettier seems to be missing in some files, but it's not a major problem for now

**Review by Caroline-CD — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**